.github/workflows/e2e.yml

```diff
@@ -0,0 +1,80 @@
+name: Integration Testing
+
+on:
+  workflow_call:
+  workflow_dispatch:
+
+jobs:
+  test:
+    name: cs9 x86_64 e2e
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTS_DIR: exported-artifacts
+    steps:
+      - name: Install dependencies
+        run: |
+          sudo apt-get -y install podman
+
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          # Use githash of a tested commit instead of merge commit
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: rpm-artifacts
+          path: ${{ env.ARTIFACTS_DIR}}
+
+      - name: build hirte tests containers
+        run: |
+          podman build -f tests/Containerfile -t hirte-image .
+
+      - name: run hirte containers
+        run: |
+          podman run -d --net podman --rm -p 2020:2020 --name hirte-mgr \
+          $(podman images *hirte-image --format {{.ID}})
+          podman run -d --net podman --rm --name hirte-node-bar \
+          $(podman images *hirte-image --format {{.ID}})
+          podman run -d --net podman --rm --name hirte-node-foo \
+          $(podman images *hirte-image --format {{.ID}})
+          PODMAN_NET=$(hostname -I | awk '{print $1}')
+          echo "PODMAN_NET=$PODMAN_NET" >> $GITHUB_ENV
+
+      - name: run hirte orchestrator
+        run: |
+          podman exec -it hirte-mgr bash -c "man hirte.conf | \
+               grep -A 5 \"\[hirte\]\" | sed -e 's/^ *//g' \
+               -e 's/journald/stderr/' \
+               -e 's/agent.*/foo,bar/' > /etc/hirte/hirte.conf"
+          podman exec -d  hirte-mgr \
+               bash -c "hirte > mgr.log 2>&1 &"
```

Comment:
Can't we use systemctl to start services?

```
podman exec -d systemctl start hirte
```

Also wouldn't it be better to use `journactl` instead of redirecting stderr to manual log file?

Reply:
It's already included in RPM installation: https://github.com/containers/hirte/pull/181

Reply:
OK it is new, thanks